src/main/kotlin/org/rust/ide/docs/RsDocumentationProvider.kt

```diff
@@ -181,6 +180,7 @@ private val RsItemElement.declarationModifiers: List<String> get() {
         }
         is RsStructItem -> modifiers += "struct"
         is RsEnumItem -> modifiers += "enum"
+        is RsConstant -> modifiers += "const"
```

Comment:
Oops, we forgot about static here)
I'll fix it

Reply:
Thanks for the fix!